scripts/release/build.js

```diff
@@ -43,7 +43,13 @@ async function copyFiles() {
 
 function createManifest() {
   const full = require('../../package.json');
-  const minimal = { main: 'index.js' };
+  const minimal = {
+    main: 'data.json',
+    exports: {
+      '.': './data.json',
+      './NodeWrapper': './nodewrapper.mjs',
```

Comment:
I find the public facing naming a bit surprising. How about just calling it `legacy`?

```suggestion
      './legacy': './legacy.mjs',
```

Reply:
Hmm...I'm not a fan of "legacy" as the name, as it may confuse consumers to think that it's some older version of the data itself, rather than a wrapper for certain NodeJS versions.  Perhaps just "wrapper" could work?

Reply:
Or actually, what about "legacyNode", to make it clear it's for older NodeJS versions and may be removed in a future major release of BCD?

Reply:
The good thing about "legacy" is that people can kind of expect that this will go away some time, whereas "wrapper" sounds like a wrapper object or function, and "legacyNode" also sounds like we're importing a "Node" object or something.

How about `forLegacyNode` to signal "the same, but for legacy NodeJS versions"?

Reply:
Name has been changed accordingly!